src/test/regress/sql/multi_schema_support.sql

```diff
@@ -69,3 +70,301 @@ SELECT master_append_table_to_shard(1190001, 'nation_local', 'localhost', :maste
 
 -- verify table actually appended to shard
 SELECT COUNT(*) FROM "nation._'append";
+
+
+-- test shard creation when search_path is set
```

Comment:
We have two newlines between different test groups. Could you use a bit more descriptive comment for this one?